tests/amqp_pause_test.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash -x
+# Test for amqp queue listening to be paused in a paused producer
+
+echo 'making sure we are clean.  one or both of these commands may fail.'
+killall nodeos
+rm -rf var/lib/node_004
+
+echo 'starting producer and sending txns'
+tests/nodeos_high_transaction_test.py -p=1 -n=8 --dump-error-details --keep-logs -v --leave-running --amqp-address=amqp://guest:guest@127.0.0.1 --num-transactions=90000000000 > var/lib/amqp.txt 2>&1 &
+sleep 5
```

Comment:
Seems like you might need more time. Again the python tests have a cleaner way to wait on nodes to come up.